graylog2-web-interface/src/views/components/sidebar/SideBar.jsx

```diff
@@ -2,17 +2,18 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 
-import { AddWidgetButton, SearchResultOverview } from 'views/components/sidebar';
+
+import { AddWidgetButton } from 'views/components/sidebar';
 import { Icon, Spinner } from 'components/common';
 import type { ViewMetaData } from 'views/stores/ViewMetadataStore';
 
-import styles from './SideBar.css';
+import { Container, ContentOverlay, SidebarHeader, Headline, ToggleIcon, HorizontalRuler } from './Sidebar.styles.jsx';
```

Comment:
@Graylog2/frontend Same thought on imports - do we want to have a max number of imports before we want to start line breaking?

Reply:
I do not care to much. But I try to not go over 120 chars.